.circleci/config.yml

```diff
@@ -90,6 +118,11 @@ jobs:
                 name: Setup python libraries
                 command: |
                     pip3 install requests pyyaml
+            - run:
+                name: Install dependencies
+                command: |
+                    sudo apt-get update && \
+                    sudo apt-get install jq
```

Comment:
not sure if we need `jq` - no longer need the unzip dep cause it's part of the maven build image

Reply:
@inodb We need `jq` for the config of the services at localdb tests.